roles/etcd/defaults/main.yaml

```diff
@@ -10,7 +10,7 @@ r_etcd_common_embedded_etcd: false
 
 osm_etcd_image: 'registry.access.redhat.com/rhel7/etcd'
 etcd_image_dict:
-  origin: "registry.fedoraproject.org/latest/etcd"
+  origin: "registry.fedoraproject.org/f27/etcd"
```

Comment:
@dustymabe Did we regress something on the fedora registry? I thought we were going to be able to rely on `registry.fedoraproject.org/latest/etcd` always pointing at the current release's build?

Reply:
added comment to global comments section